migtests/tests/analyze-schema/expected_issues.json

```diff
@@ -800,7 +800,7 @@
             "Reason": "Unsupported datatype - xml on column - data",
             "SqlStatement": "CREATE TABLE test_xml_type(id int, data xml);",
             "DocsLink": "https://docs.yugabyte.com/preview/yugabyte-voyager/known-issues/postgresql/#data-ingestion-on-xml-data-type-is-not-supported",
-            "Suggestion": "Data ingestion is not supported for this type in YugabyteDB so handle this type in different way. Refer link for more details - \u003cLINK DOC\u003e",
+            "Suggestion": "Data ingestion is not supported for this type in YugabyteDB so handle this type in different way. Refer link for more details.",
```

Comment:
even offline is not supported?

Reply:
yeah, YBDB doesn't support inserts on this type.